R/compat-plyr.R

```diff
@@ -310,6 +314,7 @@ rbind_dfs <- function(dfs) {
   attributes(out) <- list(class = "data.frame", names = names(out), row.names = .set_row_names(total))
   out
 }
+date_origin <- Sys.Date() - unclass(Sys.Date())
```

Comment:
Is this the kind of global assignment that should be moved to `.onLoad()`?

Reply:
Oh, yeah. Good catch 🙂